package.json

```diff
@@ -315,6 +315,7 @@
     "pluralize": "3.1.0",
     "pngjs": "^3.4.0",
     "polished": "^3.7.2",
+    "pretty-ms": "6.0.0",
```

Comment:
I moved this to a dependency out from a dev-dependency and bumped the version so it doesn't have the `~` in front of the text: https://github.com/sindresorhus/pretty-ms/releases/tag/v6.0.0